v63003/outref/gtm8795.txt

```diff
@@ -0,0 +1,20 @@
+##FILTERED##%YDB-I-FILERENAME, File ##TEST_PATH##/mumps.mjl is renamed to ##TEST_PATH##/mumps.mjl_##TIMESTAMP##
```

Comment:
I would like the reference file to have comments indicating what is going to be done and then include the corresponding output. Thereby reading it gives one a picture of the test path.